playground/backend/internal/utils/precompiled_objects_utils_test.go

```diff
@@ -24,7 +24,7 @@ import (
 
 func TestPutPrecompiledObjectsToCategory(t *testing.T) {
 	precompiledObjectToAdd := &cloud_bucket.PrecompiledObjects{
-		{"TestName", "SDK_JAVA/TestCategory/TestName.java", "TestDescription", pb.PrecompiledObjectType_PRECOMPILED_OBJECT_TYPE_EXAMPLE, []string{""}, "", ""},
+		{"TestName", "SDK_JAVA/TestCategory/TestName.java", "TestDescription", pb.PrecompiledObjectType_PRECOMPILED_OBJECT_TYPE_EXAMPLE, []string{""}, "", "", ""},
```

Comment:
Seems like we don't need this change then?

Reply:
Yes, changed it back.